packages/harpguru-core/src/components/harp-faces/harp-faces.tsx

```diff
@@ -4,13 +4,12 @@ import React from 'react'
 import { isChromaticHarpFace } from 'harpparts'
 
 import { HarpFace } from '../harp-face'
-import { useSizes } from '../../hooks'
 
 export const HarpFaces = (): React.ReactElement => {
   const [fullInteractionMatrix] = useGlobal('activeInteractionMatrix')
   const {
-    dynamicSizes: { fragmentGutter },
-  } = useSizes()
+    [0]: { fragmentGutter },
+  } = useGlobal('dynamicSizes')
```

Comment:
Similar point to https://github.com/js-jslog/harpguru/pull/137#discussion_r765155671

Reply:
Done